package/MDAnalysis/analysis/rms.py

```diff
@@ -332,6 +334,78 @@ def process_selection(select):
     return select
 
 
+def iterative_average(
+    mobile, ref, select='all', weights=None, niter=100, eps=1e-6,
+    verbose=False, **kwargs
+):
+    """Calculate an optimal reference that is also the average structure
+    after an RMSD alignment.
+    The optimal reference is defined as average structure of a
+    trajectory, with the optimal reference used as input.
+    This function computes the optimal reference by using a starting
+    reference for the average structure, which is used to calculate the
+    average structure again. This is repeated until the reference
+    structure has converged.
+
+    Parameters
+    ----------
+    mobile : mda.Universe
+        Universe containing trajectory to be fitted to reference.
+    ref : mda.Universe
```

Comment:
make it optional (if not set, then use mobile as reference). change the name to `reference` to match AverageStructure.